doc/api/vm.md

```diff
@@ -125,6 +125,10 @@ console.log(util.inspect(sandbox));
 // { animal: 'cat', count: 12, name: 'kitty' }
 ```
 
+*Note*: Using the `timeout` or `breakOnSigint` options will result in new
+event loops and corresponding threads being started, which have a non-zero
```

Comment:
Does using `breakOnSigint` option start a new event loop though? Either I am wrong or the language is somewhat confusing here.